zebra-consensus/src/block/subsidy/general.rs

```diff
@@ -0,0 +1,203 @@
+//! Block and Miner subsidies, halvings and target spacing modifiers. - [§7.7][7.7]
+//!
+//! [7.7]: https://zips.z.cash/protocol/protocol.pdf#subsidies
+
+use std::convert::TryFrom;
+
+use zebra_chain::{
+    amount::{Amount, Error, NonNegative},
+    block::Height,
+    parameters::{Network, NetworkUpgrade::*},
+};
+
+use super::founders_reward::founders_reward;
+use crate::parameters::subsidy::*;
+
+/// `SlowStartShift()` as described in [protocol specification §7.7][7.7]
+///
+/// [7.7]: https://zips.z.cash/protocol/protocol.pdf#subsidies
+fn slow_start_shift() -> Height {
+    Height(SLOW_START_INTERVAL.0 / 2)
+}
```

Comment:
This is a constant, so I moved it to parameters in 86a2ea5aff094e5b4f2947438182a42ed7888cc9 in oxarbitrage#29